snap/info_test.go

```diff
@@ -705,6 +705,11 @@ func (s *infoSuite) TestAppInfoIsService(c *C) {
 apps:
   svc1:
     daemon: potato
+    sockets:
+      sock1:
+        listen-stream: s1.socket
+      sock2:
+        listen-stream: s2.socket
```

Comment:
Where is this being used?  I can't see any tests leveraging that.

Reply:
Probably leftover from previous code, dropped.